murmur_test.go

```diff
@@ -113,153 +114,68 @@ func TestIncremental(t *testing.T) {
 	}
 }
 
-//---
-
-func bench32(b *testing.B, length int) {
-	buf := make([]byte, length)
-	b.SetBytes(int64(length))
-	b.ResetTimer()
-	for i := 0; i < b.N; i++ {
-		Sum32(buf)
+func Benchmark32(b *testing.B) {
+	for length := 1; length <= 8192; length *= 2 {
+		b.Run(strconv.Itoa(length), func(b *testing.B) {
+			buf := make([]byte, length)
```

Comment:
To avoid excessive allocations and triggering a GC run during the benchmark, my approach is to have a single large buffer and reslice it to the appropriate size, rather than allocating a new one each time.

Reply:
Done.